src/main/java/com/olgaboiar/mint/handlers/RouteHandler.java

```diff
@@ -1,16 +1,12 @@
 package com.olgaboiar.mint.handlers;
 
-import com.olgaboiar.mint.Request;
-import com.olgaboiar.mint.Response;
-import com.olgaboiar.mint.ResponseGenerator;
+import com.olgaboiar.mint.*;
 
 import java.io.IOException;
 
-import static com.olgaboiar.mint.Constants.STATUS_CODE_200;
-
 public class RouteHandler implements IHandler {
     @Override
-    public Response handleRequest(Request request) throws IOException {
-        return new ResponseGenerator().generateResponse(STATUS_CODE_200, "");
+    public Response handleRequest(Request request, IRouteMap routes) throws IOException {
+        return new ResponseGenerator().generateResponse(Constants.Status.STATUS_CODE_200.toString(), "");
```

Comment:
Since the ResponseGenerator does not have any state, rather than create a new instance of ResposneGenerator each time, try change #generateResponse into a static method. Let me know if you have any question on this one.

Reply:
changed in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/98d22123cb9492c165d373c18063df5273353bd5